docs/pages/machine-id/troubleshooting.mdx

```diff
@@ -0,0 +1,286 @@
+---
+title: Machine ID Troubleshooting Guide
+description: Troubleshooting common issues with Machine ID
+---
+
+## A bot failed to renew a certificate due to a "generation mismatch", what does this mean?
+
+### Symptoms
+
+The bot will log an error like this:
+
+```text
+ERROR: renewable cert generation mismatch: stored=3, presented=2
+```
+
+Subsequent connection attempts by the bot may see errors like the following:
+```text
+ERROR: failed direct dial to auth server: auth API: access denied [00]
+"\tauth API: access denied [00], failed dial to auth server through reverse tunnel: Get \"https://teleport.cluster.local/v2/configuration/name\": Get \"https://example.com:3025/webapi/find\": x509: cannot validate certificate for example.com because it doesn't contain any IP SANs"
+"\tGet \"https://teleport.cluster.local/v2/configuration/name\": Get \"https://example.com:3025/webapi/find\": x509: cannot validate certificate for example.com because it doesn't contain any IP SANs"
+```
+
+In particular, note the message `auth API: access denied`. The Teleport auth
+server will provide some additional context:
+```text
+[AUTH]      WARN lock targeting User:"bot-example" is in force: The bot user "bot-example" has been locked due to a certificate generation mismatch, possibly indicating a stolen certificate. auth/apiserver.go:224
```

Comment:
Is this a log on the Machine ID host? Otherwise, we might want to use a `ScopedBlock` to hide the message about this log for Cloud users.

Reply:
Ah, good catch, I'll add a `ScopedBlock`